Makefile

```diff
@@ -87,7 +90,7 @@ install: libdragon.a libdragonsys.a
 	install -Cv -m 0644 include/libdragon.h $(INSTALLDIR)/mips64-elf/include/libdragon.h
 	install -Cv -m 0644 include/ucode.S $(INSTALLDIR)/mips64-elf/include/ucode.S
 
-clean:
+clean: install-mk
```

Comment:
This one isn't really required, is it?

Reply:
It is not required for the clean recipe but for `examples-clean` as the child makefiles does not have the file before anything is installed.